_posts/2020-05-05-churn-vs-complexity-vs-coverage.markdown

```diff
@@ -0,0 +1,255 @@
+---
+layout: post
+title: "Churn vs. Complexity vs. Code Coverage"
+date: 2020-05-05 2:25:00
+categories: ["code-quality"]
+author: etagwerker
+---
+
+Churn vs. Complexity analysis is a great way to find insights about the maintainability
+of a project. Two of my favorite authors have written great articles about
```

Comment:
@etagwerker any reason why you're breaking the sentences into multiple lines?

Reply:
@lubc I like to keep lines under 80 characters to make it easier to see diffs.